### PR TITLE
change(web): adds error + console logs usable for Sentry reporting targeting 🍒

### PR DESCRIPTION
**Pull request by jahorton**
A :cherries:-pick of #4821.

### CODE REVIEW

**Review by darcywong00 — APPROVED** [review]
lgtm

File: web/source/dom/domEventHandlers.ts [hunk 1]
```diff
@@ -791,10 +799,8 @@ namespace com.keyman.dom {
       }
       
       // Identify the input element from the touch event target (touched element may be contained by input)
-      if(target.className == null || target.className.indexOf('keymanweb-input') < 0) target=<HTMLElement> target.parentNode;
-      if(target.className == null || target.className.indexOf('keymanweb-input') < 0) target=<HTMLElement> target.parentNode;
-      if(target.className == null || target.className.indexOf('keymanweb-input') < 0) return;
-      
+      target = findTouchAliasTarget(target);
```
Comment (mcdurdin) [inline web/source/dom/domEventHandlers.ts:802]: I guess technically `findTouchAliasTarget` can return `null` (even if that is circumvented with a throw for logging purposes at present) so we should probably handle that case with a bare `return` as in the old code... Missed that last time!

File: web/source/dom/touchAliasElement.ts [hunk 2]
```diff
@@ -30,26 +30,83 @@ namespace com.keyman.dom {
...
+                childIndex = i;
+                break;
+              }
+            }
+          }
+
+          // ID:  should never be specified for TouchAliasElement components
+          // Class:  specified for the root TouchAliasElement and for its text spans
+          // index:  useful for identifying elements with no class info
+          return `tag type ${element.tagName}, class '${element.classList}', index under parent: '${childIndex}', id '${element.id}'`;
```
Comment (mcdurdin) [inline web/source/dom/touchAliasElement.ts:55]: Is `` ` `` going to cause a problem on old versions of Chrome? https://caniuse.com/?search=backtick

**Review by mcdurdin — APPROVED** [review]
LGTM I think. Surprising that I missed these two questions in the last PR but ...

File: web/source/dom/touchAliasElement.ts (hunk 2 above)
Comment (jahorton) [reply web/source/dom/touchAliasElement.ts:55]: It won't.  The compiled version of this line, as seen from the unminified build product:

```javascript
// ID:  should never be specified for TouchAliasElement components
// Class:  specified for the root TouchAliasElement and for its text spans
// index:  useful for identifying elements with no class info
return "tag type " + element.tagName + ", class '" + element.classList + "', index under parent: '" + childIndex + "', id '" + element.id + "'";
```

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.272](https://keyman.com/downloads/releases/stable/14.0.272)

**Pull request merged**